src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -203,6 +210,45 @@ def _migrate_dbfs_root_table(self, src_table: Table, rule: Rule, grants: list[Gr
         self._backend.execute(src_table.sql_alter_from(rule.as_uc_table_key, self._ws.get_workspace_id()))
         return self._migrate_acl(src_table, rule, grants)
 
+    def _migrate_non_sync_table(self, src_table: Table, rule: Rule, grants: list[Grant] | None = None):
+        table_migrate_sql = self._get_create_in_place_sql(src_table, rule)
+        logger.debug(f"Migrating table (No Sync) {src_table.key} to using SQL query: {table_migrate_sql}")
+        self._backend.execute(table_migrate_sql)
+        self._backend.execute(src_table.sql_alter_to(rule.as_uc_table_key))
+        self._backend.execute(src_table.sql_alter_from(rule.as_uc_table_key, self._ws.get_workspace_id()))
```

Comment:
@FastLee : Why does the src_table get both the upgraded "from" and "to" tbl property? I would expect only the "upgraded to" to be applied for the source table

Reply:
The source table holds the SQL to upgrade both the from and to table.
Nice observation.